photoUtil.js

```diff
@@ -0,0 +1,7 @@
+export const isUserTagged = (user, photo) => {
+  return photo.taggedUsers.map(user => user.id).includes(user.id);
+};
+
+export const isUserBlocked = (user, photo) => {
+  return photo.user.blockedFriends.map(user => user.id).includes(user.id);
+};
```

Comment:
These 2 functions contain similar code. I'm okay if you leave it like that. If you want to refactor it a bit, you extract the similar part to:
```
const isUserIn = (user, userPool) => {
  return userPool.map(user => user.id).includes(user.id);
}
```
and use it like this:
```
export const isUserTagged = (user, photo) => {
  return isUserIn(user, photo.taggedUsers);
}

export const isUserTagged = (user, photo) => {
  return isUserIn(user, photo.user.blockedFriends);
}
```